README.md

```diff
@@ -1,11 +1,17 @@
 [![Build Status](https://travis-ci.org/OCA/margin-analysis.svg?branch=8.0)](https://travis-ci.org/OCA/margin-analysis)
 [![Coverage Status](https://img.shields.io/coveralls/OCA/margin-analysis.svg)](https://coveralls.io/r/OCA/margin-analysis?branch=8.0)
 
-Margin Analysis
-===============
+Margin Analysis / Financial controlling
```

Comment:
In the same time you can make the title a real title by adding some '====' above the module title. :smirk: